packages/api-explorer/src/scenes/TypeScene/TypeScene.tsx

```diff
@@ -40,26 +40,32 @@ import {
   ExploreType,
   DocSchema,
 } from '../../components'
+import { navigate } from '../../utils/navUtils'
 
 interface DocTypeProps {
   api: ApiModel
 }
 
 interface DocTypeParams {
   specKey: string
+  typeTag: string
   typeName: string
 }
 
 export const TypeScene: FC<DocTypeProps> = ({ api }) => {
-  const { specKey, typeName } = useParams<DocTypeParams>()
+  const { specKey, typeTag, typeName } = useParams<DocTypeParams>()
```

Comment:
Addresses a new bug that currently occurs in the main branch as well -> e.g. clicking ApiAuth>Login and switching to the Types tab will result in the URL changing to '/3.1/types' even when the ApiAuth panel is open. URL should be '/3.1/types/ApiAuth' in this case.